scripts/prepare-test-verifiables.js

```diff
@@ -32,6 +32,30 @@ const opts = {
                 }`,
                 replace: "%presentation%"
             }
+        },
+        {
+            name: "digitalbazaar",
+            types: ["PermanentResidentCard"],
+            vc: {
+                path: "https://issuer.interop.digitalbazaar.com/credentials/issueCredential",
+                request: `{
+                "credential": %credential%
+                    }`,
+                replace: "%credential%"
+            }
+        },
+        {
+            name: "danubetech",
+            types: ["UniversityDegreeCredential", "PermanentResidentCard"],
+            vc: {
+                path: "https://uniissuer.io/danubetech/credential-issuer/0.0.1/credentials/issueCredential",
+                issuer: "did:v1:test:nym:z6Mkeac33uiSWFAfDM5wPUWDhGZFUWDvTji3zJ1yg4ADUpiW",
```

Comment:
did:sov?

Reply:
I used example issuer from their guide, danube tech uses did:v1 too,

Changed it to 'did:dov' in this PR
https://github.com/trustbloc/edge-service/pull/198